src/server/xtra2.c

```diff
@@ -107,7 +116,7 @@ bool set_blind(int Ind, int v)
 	if (!notice) return (FALSE);
 
 	/* Disturb */
-	if (p_ptr->disturb_state) disturb(Ind, 0, 0);
+	if (option_p(p_ptr,DISTURB_STATE)) disturb(Ind, 0, 0);
```

Comment:
This line is repeated a LOT, including the lack of space after the comma. Function? Look at the patterns the code makes. There are little blocks off stuff being done, separated by these disturb() calls. If only there were some way to group these logically.